lms/djangoapps/certificates/models.py

```diff
@@ -339,9 +339,17 @@ def is_valid(self):
     def save(self, *args, **kwargs):
         """
         After the base save() method finishes, fire the COURSE_CERT_AWARDED
-        signal iff we are saving a record of a learner passing the course.
+        signal if we are saving a record of a learner passing the course.
+        As well as the COURSE_CERT_CHANGED for any save event.
         """
         super(GeneratedCertificate, self).save(*args, **kwargs)
+        COURSE_CERT_CHANGED.send_robust(
+            sender=self.__class__,
+            user=self.user,
+            course_key=self.course_id,
```

Comment:
super nit: maybe in all new code, use course_run_key instead of course_key? But parity with surrounding existing code is also useful.

Reply:
I think I'll leave it here since elsewhere in the file (and even method) it's referred to as course_key... but agreed, if I write anything new I'll be sure to use `course_run_key`